content/docs/04.6-vm-management/10-create-manage-vm/15-create-vm-template.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Create a VM Template"
+metaTitle: "Create a VM Template"
+metaDescription: "Learn how to"
+icon: " "
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Although Palette provides out-of-the box templates, we recommend that you create and manage your own templates.
+
+# Prerequisites
+
+- Valid YAML that defines your VM template(s).
+
+# Enablement
+
+Create a template by adding a YAML file as a manifest in an add-on profile.
+
+<br />
+
+1. From the left **Main Menu** click **Profiles** and click the **Add Cluster Profile** button.
+
+
+2. Give the profile a name, select type **Add-on**, and click **Next**.
+
+
+3. On the Profile Layers page, click **Add Manifest**. 
+
+
+4. Give the layer a name, and click **Edit manifest** and enter a name for the first template. Click the checkmark icon.
+
+
+5. In the blank manifest file at right, enter the VM definition as a YAML file. You can add multiple manifests for multiple templates in the same add-on profile. They will display as layers in the profile.
+
+
+6. Click **Confirm and Create**, then click **Next**.
+
+
+7. Click **Finish Configuration**. 
+
+
+
+#### Example YAML for a VM template
+
+```yaml
+apiVersion: spectrocloud.com/v1
+kind: VmTemplate
+metadata:
+  labels:
+    app.kubernetes.io/name: fedora-36
+    app.kubernetes.io/instance: fedora-36-instance
+    app.kubernetes.io/part-of: vmtemplate
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: vmtemplate
+  name: fedora-36
+spec:
+  description: This is Fedora 36 image
+  displayName: Fedora 36
+  icon: "https://s3.amazonaws.com/manifests.spectrocloud.com/logos/fedora.png"
+  running: false
+  template:
+    spec:
+      domain:
```

Comment:
@tiwarisumit3 we may want to add `evictionStrategy: LiveMigrate` to all of our templates.

Reply:
Good point @TylerGillson. @ritawatson  in the sample let's add the `evictionStrategy` flag as mentioned by Tyler

Reply:
@tiwarisumit3 @TylerGillson can you guys show us where in the YAML that flag would go please?